packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -216,6 +217,33 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
     );
   }
 
+  Future<int?> _getMyCount(
+    final String userId,
+    final UserProductSearchType type,
+    final OpenFoodFactsLanguage language,
+  ) async {
+    final UserProductSearchQueryConfiguration configuration =
+        UserProductSearchQueryConfiguration(
+      type: type,
+      userId: userId,
```

Comment:
Please put directly `OpenFoodAPIConfiguration.globalUser!.userId` and remove the method parameter.